config/event-projector.php

```diff
@@ -60,4 +68,13 @@
      * a chunked way. You can specify the chunk size here.
      */
     'replay_chunk_size' => 1000,
+
+    /*
+     * In production, you likely do not want the package to scan all the event handlers
+     * on every request. The package can cache all registered event handlers.
+     * More info: XXX
+     *
+     * Here you can specify where the cache should be stored.
+     */
+    'cache_path' => storage_path('app/event-projector/event-handlers.php'),
```

Comment:
Maybe set up a storage _directory_ so we can reuse the cache path for other features too? (I could see us using this for the auto event mapping that was recently addd)

Reply:
Done